Software/WinIOT/Readme.MD

```diff
@@ -0,0 +1,23 @@
+# BrickPi3
+## Windows 10 IoT Core implementation for the BrickPi3.  These libraries can be run with the Raspberry Pi 2 or Raspberry Pi 3.
+
+The BrickPi3 was ported to C# [by Laurent Ellerbach](https://github.com/Ellerbach/).  There is a separate repository on Github with the code and [instructions on getting started here](https://github.com/Ellerbach/BrickPi3).
+
```

Comment:
By directing users to the [BrickPi3 WinIOT repo](https://github.com/Ellerbach/BrickPi3), the remainder of the readme becomes duplicated, and will likely become depreciated.